lms/envs/aws.py

```diff
@@ -188,6 +188,8 @@
 EDXMKTG_LOGGED_IN_COOKIE_NAME = ENV_TOKENS.get('EDXMKTG_LOGGED_IN_COOKIE_NAME', EDXMKTG_LOGGED_IN_COOKIE_NAME)
 EDXMKTG_USER_INFO_COOKIE_NAME = ENV_TOKENS.get('EDXMKTG_USER_INFO_COOKIE_NAME', EDXMKTG_USER_INFO_COOKIE_NAME)
 
+LMS_ROOT_URL = ENV_TOKENS.get('LMS_ROOT_URL')
```

Comment:
@pomegranited Yes, this is the one https://github.com/edx/configuration/pull/3256; it has been merged to master though.